plugin/tiff/src/main/java/it/geosolutions/imageioimpl/plugins/tiff/TIFFLZWDecompressor.java

```diff
@@ -233,84 +211,7 @@ public int decode(byte[] sdata, int srcOffset,
 	    }
 	}
 
-        if (predictor == BaselineTIFFTagSet.PREDICTOR_HORIZONTAL_DIFFERENCING) {
```

Comment:
And this indeed is covered inside the PredictorDecompressor.decompress method, which also adds support for the floating point predictor.